test/__snapshots__/docker-compose.spec.js.snap

```diff
@@ -1641,27 +1659,6 @@ Launch all your infrastructure by running: \`docker-compose up -d\`.
       - SERVER_PORT=80
     ports:
       - \\"8080:8080\\"
-
-  jhipster-registry:
-    image: jhipster/jhipster-registry:v6.8.0
-    volumes:
-      - ./central-server-config:/central-config
```

Comment:
With the test description, there is no way to know if it's correct.

Reply:
the pseudo and password are not filled compared to the snapshot that need eureka: looking at the 'configure' method of the docker compose generator, it's meaning the eureka is not selected.
Also, looking at the test, eureka is not mentionned in the yo-rc and configuration, and by default (according to the doc), serviceDiscovery is set to none